<!--REQUEST_ID=ae3f8768-0c73-49c7-859d-9df451f43723-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47162&attempt_id=18b31a23-b087-427b-b190-699af723c435)

Attempted: php71, php72

<details><summary>Partial log (click to expand)</summary><p>

```
/private/tmp/nix-build-php-7.1.21.drv-0/php-7.1.21/main/php_config.h:2548:24: note: expanded from macro 'zend_finite'
#define zend_finite(a) finite(a)
                       ^
/nix/store/qjmqrzk8nhn5maa093fhl3finczz72lw-libc++-5.0.2/include/c++/v1/math.h:439:1: note: 'isfinite' declared here
isfinite(_A1 __lcpp_x) _NOEXCEPT
^
2 errors generated.
make: *** [Makefile:1171: ext/intl/common/common_enum.lo] Error 1
builder for '/nix/store/w0yfdlaydqk3xka0qzg3145rahp6qlyh-php-7.1.21.drv' failed with exit code 2
error: build of '/nix/store/w0yfdlaydqk3xka0qzg3145rahp6qlyh-php-7.1.21.drv' failed
```
</p></details>

--- a/pkgs/development/interpreters/php/default.nix
+++ b/pkgs/development/interpreters/php/default.nix
@@ -226,13 +226,13 @@ in {
   # We can't update the darwin versions because they simply don't compile at
   # all due to a bug in the intl extensions.
   #
-  # The bug so far is present in 7.1.21, 7.2.9, 7.2.10.
+  # The bug so far is present in 7.1.21, 7.1.22, 7.2.9, 7.2.10.
 
   php71 = generic (
     if stdenv.isDarwin then
       {
-        version = "7.1.21";
-        sha256 = "104mn4kppklb21hgz1a50kgmc0ak5y996sx990xpc8yy9dbrqh62";
+        version = "7.1.20";
+        sha256 = "0i8xd6p4zdg8fl6f0j430raanlshsshr3s3jlm72b0gvi1n4f6rs";
       }
     else
       {

